pkg/storegateway/series_chunks_test.go

```diff
@@ -175,7 +175,7 @@ func TestSeriesChunksSeriesSet(t *testing.T) {
 		it := newSeriesChunksSeriesSet(source)
 
 		lbls, chks := it.At()
-		require.Zero(t, lbls)
+		require.True(t, lbls.IsEmpty())
```

Comment:
I guess `require.Zero()` would still work, as zero value of `labels.Labels` is empty, right?